workflow/rocoto/gfs_cycled_xml.py

```diff
@@ -24,6 +24,17 @@ def get_cycledefs(self):
         sdate_str = sdate.strftime("%Y%m%d%H%M")
         strings.append(f'\t<cycledef group="gdas">{sdate_str} {edate_str} {interval_str}</cycledef>')
 
+        if self._app_config.do_jedisnowda:
+            sdate_snocvr = self._base['SDATE']
+            edate_snocvr = self._base['EDATE']
+            interval_snocvr = to_timedelta('24H')
+            sdate_snocvr = sdate_snocvr + interval_snocvr
+            sdate_snocvr_str = sdate_snocvr.replace(hour=0, minute=0, second=0).strftime("%Y%m%d%H%M")
+            edate_snocvr_str = edate_snocvr.strftime("%Y%m%d%H%M")
+            interval_str = timedelta_to_HMS(interval_snocvr)
+            if sdate_snocvr <= edate_snocvr:
+                strings.append(f'\t<cycledef group="gdas_prep_snocvr">{sdate_snocvr_str} {edate_snocvr_str} {interval_str}</cycledef>')
+
```

Comment:
The issue appears to be here.  For the C96C48_hybatmaerosnowDA case, `sdate_snocvr` and `edate_snocvr`  are initially set to (at line 28-9) `2021122012` and `2021122100`, respectively.  But then `sdate_snocvr` is reevaluated at line 31 to `2021122112`.  Thus, `sdate_snocvr` is greater than `edate_snocvr` at line 35, so the `gdas_prep_snocvr` cycledef is never defined.

Reply:
You are correct. Because your CI test is set less than 24 hour, I will try to work on this. Do you have any suggestions? Thank you very much.

Reply:
It looks like the idea is that the snow cover job should not run the first three full cycles, but on the first 00Z cycle thereafter, is that correct?  If so, then either the CI test would need to be extended to cover that period.  Also, if that is the case, then the `snowanl` job should not be looking for the `ims_snow` files the first few cycles, since they won't be generated.

If the snow cover jobs should run on the first full cycle, then this cycledef should be modified.  I would need to think about how to write that so it would run on the first full cycle and every 00Z cycle thereafter.

Reply:
We would like to include the `gdas_prepsnowcover` job on the first full cycle (only at the 00Z) and every 00Z cycle thereafter.